### DISCUSSION TITLE
Are these benchmarks set up properly?

### DISCUSSION THREAD

**Discussion by MatthewCaseres** [lifelib-dev/lifelib#50] [Q&A]
I want to make sure that I am running BasicTerm_M properly in the benchmarks repository - https://github.com/actuarialopensource/benchmarks/blob/main/Python/basicterm.py

I clear the cache by setting a variable in the model.

**Comment by fumitoh** [comment]
Yes, that's working. Calling `m.Projection.clear_all()` does the same.
